test/libsyncengine/reconciliation/platform_inconsistency_checker/testplatforminconsistencycheckerworker.cpp

```diff
@@ -154,6 +165,78 @@ void TestPlatformInconsistencyCheckerWorker::testNameClash() {
 #endif
 }
 
+void TestPlatformInconsistencyCheckerWorker::testNameClashAfterRename() {
+    // Create local files
+    std::vector<std::string> nodes = {{"a1"}, {"A"}};
+    for (const auto &n : nodes) {
+        std::ofstream ofs(_tempDir.path / n);
+        ofs << "Some content.\n";
+        ofs.close();
+    }
+
+    // Set up DB
+    DbNode dbNode_A(2, _syncPal->_syncDb->rootNode().nodeId(), Str("A"), Str("A"), "lA", "rA", defaultTime, defaultTime,
+                    defaultTime, NodeType::NodeTypeDirectory, defaultSize, std::nullopt);
+    DbNode dbNode_a(3, _syncPal->_syncDb->rootNode().nodeId(), Str("a1"), Str("a1"), "la", "ra", defaultTime, defaultTime,
+                    defaultTime, NodeType::NodeTypeDirectory, defaultSize, std::nullopt);
+    DbNodeId dbNodeIdDir_A;
```

Comment:
We usually avoid underscores and write variable names with camelCase.

Reply:
Done